compiler/rustc_attr_data_structures/src/attributes.rs

```diff
@@ -38,10 +38,20 @@ pub enum InstructionSetAttr {
 #[derive(Clone, Encodable, Decodable, Debug, HashStable_Generic)]
 pub enum OptimizeAttr {
     None,
```

Comment:
Probably better to rename it to `Default` still; to avoid the confusion with `DoNotOptimize` (if you do, please create a separate commit)

Reply:
Whoops, missed that change